app/tests/components_tests/stubs.py

```diff
@@ -50,6 +51,20 @@ def _copy_to_output(self):
                 f.write(json.dumps(res))
 
 
+class CloudwatchClientStub:
+    def get_metric_statistics(self, **__):
+        return {
+            "Label": "BurstCreditBalance",
+            "Datapoints": [
+                {
+                    "Timestamp": "2022-03-09T14:38:00+00:00",
+                    "Maximum": 2199000000000,  # Be careful changing this value, could create a lot of data
```

Comment:
Maybe use the more readable notation?
```suggestion
                    "Maximum": 2_199_000_000_000,  # Be careful changing this value, could create a lot of data
```

Reply:
Agreed, also made it a variable as if someone changed the target you may end up with a huge file generated on disk in the tests 😱